Pinetree/wwwroot/js/emoji-picker.js

```
@@ -0,0 +1,717 @@
+        '😚': ['kiss', 'love', 'closed', 'eyes'],
+        '😙': ['kiss', 'love', 'whistle'],
+        '🥲': ['tear', 'happy', 'sad', 'emotional'],
+        '😋': ['yum', 'delicious', 'tongue', 'taste'],
+        '😛': ['tongue', 'playful', 'silly'],
+        '😜': ['tongue', 'wink', 'playful', 'silly'],
+        '🤪': ['crazy', 'wild', 'silly', 'zany'],
+        '😝': ['tongue', 'silly', 'playful', 'closed', 'eyes'],
+        '🤑': ['money', 'rich', 'dollar', 'greedy'],
+        '🤗': ['hug', 'embrace', 'love', 'care'],
+        '🤭': ['oops', 'secret', 'whisper', 'giggle'],
+        '🤫': ['shush', 'quiet', 'secret', 'silence'],
```

Biome (1.9.4)

[error] This property value named 🤗 is later overwritten by an object member with the same name.

Overwritten with this value.

If an object property with the same name is defined multiple times (except when combining a getter with a setter), only the last definition makes it into the object and previous definitions are ignored.
Unsafe fix: Remove this property value named 🤗


(lint/suspicious/noDuplicateObjectKeys)